docker/Dockerfile

```diff
@@ -151,7 +151,7 @@ RUN \
 apt-get update && apt-get -y install python-pip \
 && pip --version \
 && apt-get -y install build-essential python-dev \
-&& pip install mkdocs && mkdocs --version \
+&& pip install mkdocs==0.17.4 && mkdocs --version \
 && pip install mkdocs-material
```

Comment:
shall we find the compatible mkdocs-material version for mkdocs 0.17.4 and hardcode that as well?

Reply:
Added from https://github.com/wso2/build-automation-artifacts/pull/51/commits/5353eb29283469a436e2e1390c50be2cc29d8fb5